torch/lib/c10d/UnixSockUtils.hpp

```diff
@@ -0,0 +1,92 @@
+#pragma once
```

Comment:
would I be correct if I assume contents in this file was extracted from `c10d/Utils.cpp`? Are there any changes?